### PR TITLE
Better icon matching

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Some company names are `lyft_pdx` instead of just `Lyft`. This PR matches based on the first 4 characters to hopefully match more company icons. Sadly this is nearly impossible to test because of how convoluted the otp-ui imports are.

### CODE REVIEW

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Can you split into one PR for geocoder and one for icons please?

**Comment by miles-grant-ibigroup** [comment]
Sorry, I accidentally based this pr on https://github.com/opentripplanner/otp-ui/pull/660

**Comment by miles-grant-ibigroup** [comment]
This PR is not urgent so I will have this be blocked by #660

**Comment by miles-grant-ibigroup** [comment]
PR is updated, ready for review @binh-dam-ibigroup

File: packages/icons/src/companies/index.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ const companyLookup = {
 };
 
 function getCompanyIcon(name) {
-  const icon = companyLookup[name.toLowerCase()];
+  const icon =
+    companyLookup[name.toLowerCase()] ||
+    companyLookup[name.toLowerCase().slice(4)];
```
Comment (binh-dam-ibigroup) [inline packages/icons/src/companies/index.js:25]: Add the description of the PR as a comment here, so that we'll remember where this logic came from.

**Review by binh-dam-ibigroup — APPROVED** [review]
Please add the indicated comment before proceeding.

File: packages/icons/src/companies/index.js [hunk 2]
```diff
@@ -20,7 +20,14 @@ const companyLookup = {
...
 function getCompanyIcon(name) {
-  const icon = companyLookup[name.toLowerCase()];
+  const icon =
+    companyLookup[name.toLowerCase()] ||
+    /*
+    Some company names are lyft_pdx instead of just Lyft. 
+    This PR matches based on the first 4 characters to hopefully 
+    match more company icons.
+    */
+    companyLookup[name.toLowerCase().slice(4)];
```
Comment (AdrianaCeric) [inline packages/icons/src/companies/index.js:30]: wouldn't we want to do `.slice(0,4)` instead to extract "lyft" from "lyft_pdx" or do we want "_pdx"

**Review by AdrianaCeric — COMMENTED** [review]
one question -- i might not be understanding what should be sliced

File: packages/icons/src/companies/index.js (hunk 2 above)
Comment (miles-grant-ibigroup) [reply packages/icons/src/companies/index.js:30]: Amazing catch thank you!

**Review by AdrianaCeric — APPROVED** [review]
Awesome!

**Pull request merged**